skynet/dgpu/daemon.py

```diff
@@ -73,29 +67,24 @@ async def maybe_serve_one(
         logging.warning('model not blacklisted!, skip...')
         return
```

Comment:
might want to change log message to `model is blacklisted!, skip...`

Reply:
Done